### ISSUE TITLE
Possible Accrescent App Store Availability 

### ISSUE THREAD

**Issue by NorthernMagic** [beemdevelopment/Aegis#1271]
Hello! Would it be possible for you guys to publish Aegis Authenticator on the Accrescent App Store for Android alongside the other distribution channels?

Many users, especially on privacy and security focused Android Oses like GrapheneOS or those simply privacy conscious use Accrescent as a way to download their apps securely without relying on Google services.

It would be nice for you guys to make this amazing app available there for Accrescent's users.

[https://accrescent.app/](https://accrescent.app/)

**Comment by alexbakker** [comment]
Accrescent is currently in early alpha. Let's wait for a stable release before considering spending time on this. 

It'd be helpful if the link from the FAQ to their app publishing documentation didn't 404: https://accrescent.app/docs/guide/publish.html.

**Comment by kydauran** [comment]
> Accrescent is currently in early alpha. Let's wait for a stable release before considering spending time on this.
> 
> It'd be helpful if the link from the FAQ to their app publishing documentation didn't 404: https://accrescent.app/docs/guide/publish.html.

Actually you need to ask to the dev to get invited in order to publish it (as its, as you said, in alpha)
Sometimes it can take some time.
But it would be great to see your app there !

**Comment by Ksosbjsnsks** [comment]
I would love to see you on Accrescent too! Aegis is one of the few Apps I get from Obtainium and it would be a lot easier to just get it from Accescrent. Even if it is in Alpha or Beta, I think it is worth. Because it will only grow and become better, when more people join and engage.

**Comment by PrivacyAndSecurity** [comment]
I agree with him, that we would love to see Aegis on Accrescent! Maybe you ask why we want this or what Benefits it would have for us as Users.

The Benefits for me would be a few, like these:

- I could get Aegis from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

- It would make it a lot easier for me and save me time, because Aegis is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

- Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

I really hope that you Guys could make that possible, because it would have great Benefits for us as Users. And think like that: If we support them by adding Apps, we help them to grow from Alpha to Stable.

**Comment by ghost** [comment]
I'd love to see Aegis on Accrescent. The Accrescent team is currently [working](https://blog.accrescent.app/posts/road-behind-road-ahead/#app-publishing-api) on an API for developers to automatically upload new releases of their apps to Accrescent without manual intervention. Hopefully, this opens your mind more to including Aegis on Accrescent.
